plugins/openshift/skills/generating-ovn-topology/scripts/analyze_placement.py

```
@@ -0,0 +1,299 @@
+    ROUTER_PLACEMENT_FILE = "ovn_router_placement.txt"
+
+    # Component names for visualization overrides
+    TRANSIT_SWITCH_NAME = "transit_switch"
+    JOIN_SWITCH_NAME = "join"
+
+    # Placement types
+    PLACEMENT_PER_NODE = "per-node"
+    PLACEMENT_CLUSTER_WIDE = "cluster-wide"
+    PLACEMENT_CLUSTER_WIDE_VISUAL = "cluster-wide-visual"
+
+    def __init__(self, tmpdir: str = "/tmp"):
```

Ruff (0.14.4)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)